rpm/qxcompositor.spec

```diff
@@ -4,13 +4,18 @@ Name:       qxcompositor
 %{!?qtc_qmake5:%define qtc_qmake5 %qmake5}
 %{!?qtc_make:%define qtc_make make}
 %{?qtc_builddir:%define _builddir %qtc_builddir}
-Summary:    My Sailfish OS Application
-Version:    0.0.5
-Release:    1
+Summary:    Qml compositor for running Xwayland on Sailfish
+Version:    0.0.6
+Release:    0
 Group:      Qt/Qt
-License:    LICENSE
-URL:        http://example.org/
+License:    BSD-3-Clause
+URL:        https://github.com/elros34/qxcompositor
 Source0:    %{name}-%{version}.tar.bz2
+
+# Segregate the SFOS releases range covered by each release branch (branch qtwayland-5.6 for SFOS >= 4.2.0):
+Requires: sailfish-version >= 4.2.0
```

Comment:
I can merge it if you want but this is not really needed. I see no point doing this, you just build 5.6 branch for 4.2,4.3, and so on releases and 5.4 branch for older sfos releases. Have you seen this in some other packages?